### PR TITLE
Apply resources specified in values files

### PR DESCRIPTION
**Pull request by CodeReaper**

### CODE REVIEW

File: chart/templates/node.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@ spec:
         - name: node
           image: {{ $registry }}/concordium/{{ .Values.network }}-node:{{ $.Values.image.tag | default $.Chart.AppVersion }}
           command: ["/concordium-node"]
-          {{- include "common.limits" $.resources | nindent 10 }}
+          {{- include "common.limits" $.Values.resources | nindent 10 }}
```
Comment (sahma19) [inline chart/templates/node.yaml]: What is the guarantee that this will work?

Comment (sahma19) [reply chart/templates/node.yaml]: Should i try adding a test for regression, or will i just end up testing how the templating engine works?

Comment (CodeReaper) [reply chart/templates/node.yaml]: I looked for current tests related to templating, but found none. It makes sense to me to have some tests ala given values file contents that templated outputs reflects those values, but I did not think I should introduce such tests.

**Comment by CodeReaper** [comment]
Handled in #3

**Pull request closed without merging**